site/reference/live/status/README.md

```diff
@@ -1,111 +1,81 @@
 ---
-title: "Status"
+title: "`status`"
```

Comment:
@phanimarupaka are we changing this command to be a verb i.e. `observe`? Same question for `tree`, there are the only two noun commands.